agent/cmd/checkPermission.go

```diff
@@ -105,8 +108,45 @@ var checkPermissionCmd = &cobra.Command{
 			os.Exit(-1)
 		}
 
+		keyFingerprint, err := cmd.Flags().GetString("key-fingerprint")
+		if err != nil {
+			log.WithFields(logrus.Fields{
+				"event":  "reading flag parameter from sshd",
+				"topic":  "key_fingerprint not informed",
```

Comment:
What do you think of using `key-fingerprint` instead of `key_fingerprint` here? 🙃

Reply:
Ok, this was a mistake. Corrected!